src/main/java/data/GenericData.java

```diff
@@ -456,4 +459,59 @@ public static JSONObject getRestrictions(String testCode) {
         }
         return actualRestrictions;
     }
+
+    public static String updateJson(TestResultsSteps testResultsSteps, String jsonFileName, String $) {
+        // read post request body from file
+        String body = GenericData.readJsonValueFromFile(jsonFileName,"$");
+
+        //generate datetime
+        DateTime currentTimestamp = DateTime.now().withZone(DateTimeZone.UTC);
+        DateTime testStartTimestamp = currentTimestamp.minusYears(1).minusHours(2);
+        DateTime endTimestamp = currentTimestamp.minusYears(1).minusHours(1);
+
+
+        String startTime = testStartTimestamp.toString();
+        String endTime = endTimestamp.toString();
+
+        String body2 =  body.replaceAll("testStartTimestamp.*?\\,", "testStartTimestamp\":\""+startTime+"\",");
+
+        String body3 =  body2.replaceAll("testEndTimestamp.*?\\,", "testEndTimestamp\":\""+endTime+"\",");
+
+        String body4 =  body3.replaceAll("testTypeStartTimestamp.*?\\,", "testTypeStartTimestamp\":\""+startTime+"\",");
+
+        String body5 =  body4.replaceAll("testTypeEndTimestamp.*?\\,", "testTypeEndTimestamp\":\""+endTime+"\",");
+
+        String finalBody = body5;
+
+        return finalBody;
+    }
+
+    public static String updateJson2(TestResultsSteps testResultsSteps, String jsonFileName, String $) {
```

Comment:
testResultSteps passed into this method is no longer being used, is it still required?